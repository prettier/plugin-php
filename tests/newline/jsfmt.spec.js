run_spec(__dirname, ["php"], { requirePragma: true, endOfLine: "cr" });
run_spec(__dirname, ["php"], { requirePragma: true, endOfLine: "crlf" });
run_spec(__dirname, ["php"], { requirePragma: true, endOfLine: "lf" });
