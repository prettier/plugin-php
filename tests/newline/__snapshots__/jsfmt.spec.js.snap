// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CRLF.php - php-verify: CRLF.php 1`] = `
<?php
$test = 1;

class Foo
{
    /**
     * Comment indented with tabs in file with utf-8[dos]
     */
    private $foo;
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<?php
$test = 1;

class Foo
{
    /**
     * Comment indented with tabs in file with utf-8[dos]
     */
    private $foo;
}

`;

exports[`LF.php - php-verify: LF.php 1`] = `
<?php
$test = 1;
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<?php
$test = 1;

`;

exports[`file-terminated-with-multiple-LF.php - php-verify: file-terminated-with-multiple-LF.php 1`] = `
<?php
$test = 1;


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<?php
$test = 1;

`;

exports[`file-terminated-without-line-ending.php - php-verify: file-terminated-without-line-ending.php 1`] = `
<?php
$test = 1;~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<?php
$test = 1;

`;

exports[`html-CRLF.php - php-verify: html-CRLF.php 1`] = `
<!DOCTYPE html>
<html lang="de">
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1" />
        <title>.</title>
    </head>
    <body>

        <?php echo 'foo!'; ?>

    </body>
</html>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<!DOCTYPE html>
<html lang="de">
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1" />
        <title>.</title>
    </head>
    <body>

        <?php echo 'foo!'; ?>

    </body>
</html>

`;

exports[`html-LF.php - php-verify: html-LF.php 1`] = `
<!DOCTYPE html>
<html lang="de">
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1" />
        <title>.</title>
    </head>
    <body>

        <?php echo 'foo!'; ?>

    </body>
</html>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<!DOCTYPE html>
<html lang="de">
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1" />
        <title>.</title>
    </head>
    <body>

        <?php echo 'foo!'; ?>

    </body>
</html>

`;
